### PR TITLE
[14.0][FW] [13.0] shopfloor_mobile: split core -> shopfloor_mobile_base

### PR DESCRIPTION
**Pull request by sebalix**
Port of #190 from 13.0 to 14.0.

Migrate/enable two modules at the same time:
- add `shopfloor_workstation_mobile`
- `shopfloor_example` (renamed from `shopfloor_mobile_custom_example` which was already in the repository)

### CODE REVIEW

File: shopfloor_workstation_mobile/static/wms/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.DS_Store
```
Comment (simahawk) [inline shopfloor_workstation_mobile/static/wms/.gitignore]: here by error I think

**Review by simahawk — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

File: shopfloor_workstation_mobile/static/wms/.gitignore (hunk 1 above)
Comment (sebalix) [reply shopfloor_workstation_mobile/static/wms/.gitignore]: Removed

**Comment by sebalix** [comment]
I included https://github.com/OCA/wms/pull/203 too from 13.0 + address your remark @simahawk

**Pull request merged**